explorer/explorertypes.go

```diff
@@ -510,10 +510,10 @@ type AddrPrefix struct {
 // AddressPrefixes generates an array AddrPrefix by using chaincfg.Params
 func AddressPrefixes(params *chaincfg.Params) []AddrPrefix {
 	Descriptions := []string{"P2PK address",
-		"P2PKH address prefix",
-		"P2PKH address prefix",
+		"P2PKH address prefix. Standard wallet address. 1 public key -> 1 private key",
+		"Ed25519 high-speed signature",
 		"secp256k1 Schnorr P2PKH address prefix",
-		"P2SH address prefix",
+		"P2SH Stake Pool (multi-sig 1-of-2) address",
```

Comment:
Stake pool is the most common, but P2SH can be any purpose.  I'm not sure if we can same much more beyond "P2SH address prefix" except perhaps to spell out "pay to script hash".